plugins/Synchro/SynchroSynth.cpp

```diff
@@ -0,0 +1,570 @@
+/*
+ * SynchroSynth.cpp - 2-oscillator PM synth
+ *
+ * Copyright (c) 2020 Ian Sannar <ian/dot/sannar/at/gmail/dot/com>
+ * Credit to @DouglasDGI "Lost Robot" for performance optimizations
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ */
+
+#include "SynchroSynth.h"
+//Standard headers
+#include <cmath>
+//QT headers
+#include <QDomDocument>
+//LMMS headers
+#include "Engine.h" //Required to access sample rate
+#include "InstrumentTrack.h" //Required for the plugin to appear in LMMS
+#include "lmms_constants.h" //Greater precision than math.h constants?
+#include "lmms_math.h" //Faster math
+#include "Mixer.h" //Required to access sample rate
+#include "NotePlayHandle.h" //Required for audio rendering
+#include "plugin_export.h" //Required for the plugin to appear in LMMS
+
+//Plugin metadata
+extern "C"
+{
+	Plugin::Descriptor PLUGIN_EXPORT synchro_plugin_descriptor =
+	{
+		STRINGIFY(PLUGIN_NAME),
+		"Synchro",
+		QT_TRANSLATE_NOOP("pluginBrowser", "2-oscillator PM synth"),
```

Comment:
The `pluginBrowser` translation context has changed to `PluginBrowser` since #5629 (just merged, so this didn't make it into the previous review).

Reply:
Changed 👍. Unfortunately it won't build anymore because this PR doesn't have the code from https://github.com/LMMS/lmms/pull/5629.

Reply:
Build error actually seems to be down to a missing open parenthesis on line 176 of `SynchroSynth.cpp`:
https://github.com/LMMS/lmms/blob/3b7b55b1318c0c98342fcfcd8c518caa32f0cc5e/plugins/Synchro/SynchroSynth.cpp#L176

Reply:
Whoops, that would make more sense. Fixed.